src/invariants/RwaTokenInvariant.sol

```diff
@@ -22,7 +22,7 @@
 
 pragma solidity 0.6.12;
 
-import "../tokens/RwaToken.sol";
+import "../tokens/rwa/RwaToken.sol";
```

Comment:
I wonder if we need this subdirectory... the entire repo is for RWA, so maybe let's just have it within `./tokensqRwaToken.sol`?

Reply:
Just though we have a `tokens` folder which means we can have multiple tokens and this factory will work only for the current one.